apps/stories/src/useMultiMediaStreams.stories.tsx

```diff
@@ -13,12 +13,7 @@ import {
   useDisclosure,
 } from '@chakra-ui/react';
 import VideoView from '@millicast-react/video-view';
-import useMultiMediaStreams, {
-  Stream,
-  StreamId,
-  StreamTypes,
-  Resolution,
-} from '@millicast-react/use-multi-media-streams';
+import useMultiMediaStreams, { Stream, StreamTypes, Resolution } from '@millicast-react/use-multi-media-streams';
```

Comment:
2 issues for this story:
- there are duplicated control buttons in the video-view and the stream
- we didn't add the local file stream story

So please create another issue in GitHub, we will fix it later.

Reply:
We will fix them in #274